renderdoc/driver/vulkan/vk_debug.cpp

```diff
@@ -1393,6 +1395,23 @@ void VulkanReplay::CreateResources()
       });
 
   CREATE_OBJECT(m_MeshFetchDescSet, m_General.DescriptorPool, m_MeshFetchDescSetLayout);
+
+  GPA_vkContextOpenInfo context{
```

Comment:
Please don't use `{}` braces here when constructing an object, instead use normal `()` brackets.

Reply:
This is a struct without any ctor.

Reply:
OK, then it should be written as `GPA_vkContextOpenInfo context = { ... }` for C style brace initialization.